CHANGELOG.md

```diff
@@ -6,6 +6,38 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
  * Drop Ruby 2.2 support (target 2.3 and up) ([#424](https://github.com/cucumber/cucumber-rails/pull/424) [mvz])
  
+ * Begin to update the core runtime / development dependencies to something a little more recent
+   ([#431](https://github.com/cucumber/cucumber-rails/pull/431) /
+   [#432](https://github.com/cucumber/cucumber-rails/pull/432) /
+   [mvz](https://github.com/mvz) / 
+   [luke-hill](https://github.com/luke-hill))
+   
+### Fixed
+
+ * Various rubocop / styling issues that had built up over time
+  ([#413](https://github.com/cucumber/cucumber-rails/pull/413) /
```

Comment:
Pull request #413 is about dependencies so doesn't belong here.

Reply:
It was mainly just doing rubocop auto-fixes. It also fixed up some dependencies yes. I'll add it as a separate entry if you'd prefer

Reply:
I'll check this again.

Reply:
I think it makes most sense to just link to this pull request with both changelog entries.